addons/hr/views/hr_employee_views.xml

```diff
@@ -293,6 +293,20 @@
             </field>
         </record>
 
+        <record id="hr_employee_tree_view" model="ir.ui.view">
```

Comment:
Let's call it `list` since now the tag is `list` and no longer `tree`

```suggestion
        <record id="hr_employee_list_view" model="ir.ui.view">
```